Sources/MarigoldSwift/Marigold+Concurrency.swift

```diff
@@ -22,20 +22,6 @@ extension Marigold {
     
     // MARK: Device
     
-    /**
-     *  Asyncronously clears any of the Attribute, Message Stream, or Event data from the device.
-     *
-     *  Use this method to clear the device attributes after user logout.
-     *
-     *  - Parameter types: A bitwise OR collection of MARDeviceDataType dictating which sets of data to clear.
-     *  - Throws: Error when call fails.
-     **/
-    public func clearDeviceData(for types: MARDeviceDataType) async throws {
```

Comment:
We shouldn't remove this method - clearDeviceData still exists. We should keep it but remove any mention of `Attributes` from the description as this has now been moved out to the `clearAttributes` method.